test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/RazorDirectivesTest/AddTagHelperDirective_InvalidLookupText_AddsError.stree.txt

```diff
@@ -1,14 +1,18 @@
-Markup block - Gen<None> - 17 - (0:0,0)
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (0:0,0) - Tokens:1
-        SyntaxKind.Unknown;[];
-    Directive block - Gen<None> - 17 - (0:0,0)
-        Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-            SyntaxKind.Transition;[@];
-        MetaCode span - Gen<None> - [addTagHelper] - SpanEditHandler;Accepts:None - (1:0,1) - Tokens:1
-            SyntaxKind.Identifier;[addTagHelper];
-        Markup span - Gen<None> - [ ] - SpanEditHandler;Accepts:None - (13:0,13) - Tokens:1
-            SyntaxKind.Whitespace;[ ];
-        Code span - Gen<AddTagHelper:{Foo;Foo;;} [RZ1036(14:0,14 [3] )]> - [Foo] - SpanEditHandler;Accepts:AnyExceptNewline - (14:0,14) - Tokens:1
-            SyntaxKind.Identifier;[Foo];
-    Markup span - Gen<Markup> - [] - SpanEditHandler;Accepts:Any - (17:0,17) - Tokens:1
-        SyntaxKind.Unknown;[];
+RazorDocument - [0..17)::17 - [@addTagHelper Foo]
+    MarkupBlock - [0..17)::17
+        MarkupTextLiteral - [0..0)::0 - [] - Gen<Markup> - SpanEditHandler;Accepts:Any
+            Marker;[];
+        CSharpCodeBlock - [0..17)::17
+            RazorDirective - [0..17)::17
+                CSharpTransition - [0..1)::1 - Gen<None> - SpanEditHandler;Accepts:None
+                    Transition;[@];
+                RazorDirectiveBody - [1..17)::16
+                    RazorMetaCode - [1..13)::12 - Gen<None> - SpanEditHandler;Accepts:None
```

Comment:
Might eventually make sense to lift this outside of the directive body since it represents the directive identifier.

Reply:
Added to https://github.com/aspnet/Razor/issues/2619#issue-364659549